src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css

```diff
@@ -16,4 +16,21 @@
 
 .summary-view__card {
     background-color: #f2f4f7;
+}
+
+.containers-select__group {
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
```

Comment:
we can use .pb-0-imp

Reply:
Actually, this is written to apply internal styling to internal group class of react select with help of classNamePrefix.